src/dbnode/server/server.go

```diff
@@ -147,6 +151,13 @@ func Run(runOpts RunOptions) {
 		os.Exit(1)
 	}
 
+	// Currently, the zapLogger is being used for tracing only.
```

Comment:
how come we have to use zap for tracing? is that just how it works?

Reply:
The thing Andrew wrote to create the tracer in m3x uses `jaegerzap.NewLogger()` - also @prateek wants to move to using a zap logger anyway.

Reply:
yeah, i'll try to get around to this soon. want all our code to use zap instead of m3/x/log

Reply:
Agreed, it is time to kill the x logger.

Reply:
PS we should really add the ZapLogger to instrument options (or pull the Zap logger from there?  Does it not already get created in instrument options by default?).